### ISSUE TITLE
Help with separation of data from same record

### ISSUE THREAD

**Issue by jdarling** [dc-js/dc.js#838]
Doing a dashboard and have run into an issue that I'm not sure how to work around.  My data is a count of the # of activities pre-aggregated down to 15 minute intervals.  Something as follows:

```
{
  key: 'POST://api/signin',
  stats: {
    count: Number(), // Total number of times this happened in 15 minutes
    qaCount: Number(), // Number of activities attributed to QA
    realCount: Number(), // Number of activities attributed to Real Users
    slowCount: Number() // Number of activities that took more than 1 second
  },
  date: 'YYYY-MM-DD', // Just the date part of an ISODate
  time: 'YYYY-MM-DDTHH:MM:SS.mmmZ' // Typical ISODate format
}
```

I have graphs that show the # of actions grouped by day based on each value above with no problem.  Now I need to add two pie charts showing the % of slow to the % of not slow transactions and one showing the % of QA to the % of not QA for a given time period.  Since these are are really the same question lets just focus on the slow one as that is what I've been playing with.

This was my first attempt:
```
// Dimension and Group
    var slowPeriod = ndx.dimension(function(d){
      return d.stats.slowCount > 0?'Slow':'Normal';
    });
    var totalsByDay = ndx.dimension(function(d){
      return d.dd;
    });

// Chart
      slowPeriodChart.width(document.body.offsetWidth*0.2)
        .height(180)
        .radius(80)
        .dimension(this.state.slowPeriod)
        .group(this.state.slowPeriodGroup)
        .label(function (d) {
          if (slowPeriodChart.hasFilter() && !slowPeriodChart.hasFilter(d.key)) {
            return d.key + '(0)';
          }
          var label = d.key;
          if (all.value()) {
            label += '(' + d.value + ')';
          }
          return label;
        });
```

This gives me a chart with values on it that can be grouped down for view easy enough, but it doesn't actually show what I want it to as its counting all of the record counts with slow times as slow against all of the records without slow times as not slow.  What I want is to see the # of slow records compared to the # of not slow records.

Basically I need to create a domain that creates two records for each record in the source one for slow's and one for normals.  Then create my group sums based on that.  I just can't figure out how.

Anyone done similar, have an example, or know of a tutorial on this?

Thanks,
 - Jeremy

**Comment by jdarling** [comment]
Oh one more thing, I'd like to be able to click on the "Slow" entry in the Pie chart and have the rest filtered down to only "slow" numbers.

**Comment by gordonwoodhull** [comment]
Hi @jdarling,

Sorry, but the issue tracker is for reporting bugs and making enhancement requests, not for general support questions. Please ask on [Stack Overflow](stackoverflow.com) with the dc.js tag, or on [the user group](https://groups.google.com/forum/?fromgroups=#!forum/dc-js-user-group), and we will be glad to help you there. 

I am closing this.

Sounds like it may be a case for [fake groups](https://github.com/dc-js/dc.js/wiki/FAQ#filter-the-data-before-its-charted), but I haven't thought it through.
